sympy/algebras/quaternion.py

```diff
@@ -547,16 +547,19 @@ def rotate_point(pin, r):
         ==========
 
         pin : tuple
-            A 3-element tuple of coordinates of a point. This point will be
-            the axis of rotation.
-        r
-            Angle to be rotated.
+            A 3-element tuple of coordinates of a point which needs to be
+            rotated.
+        r : Quaternion or tuple
+            Axis and angle of rotation.
+
+        It's important to note that when r is a tuple, it must be of the form
+        (axis, angle)
```

Comment:
I think that this should be indented.
Currently, the numpydoc extension treats any non-indented block under `Parameters` as a parameter name
![image](https://user-images.githubusercontent.com/34944973/66127530-e19fc200-e626-11e9-91a0-2e017cdc2ffc.png)

Reply:
Ok, I'll change it. Also, @sylee957 how can I check the numpydoc for any change I make in documentaion?

Reply:
Right now, there would be no other way than building the doc manually from `sympy/doc`
There is a open discussion to automate this process in #17597 though

Reply:
Oh, ok! Thanks!!